### ISSUE TITLE
Undefined name: missing imports?

### ISSUE THREAD

**Issue by cclauss** [max-dotpy/GUI_maker#1]
flake8 testing of https://github.com/max-dotpy/GUI_maker on Python 3.8.1

$ `flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics`
```
./myTkinter.py:18:21: F821 undefined name 'Image'
            image = Image.open(my_kw["img"])
                    ^
./myTkinter.py:19:21: F821 undefined name 'ImageTk'
            photo = ImageTk.PhotoImage(image)
                    ^
2     F821 undefined name 'Image'
2
```

**Comment by max-dotpy** [comment]
Thank you! It was missing
`from PIL import Image, ImageTk`

**Comment by cclauss** [comment]
Fixed in 5249f21
